src/components/sectionList/toolbar/Toolbar.tsx

```diff
@@ -16,14 +17,18 @@ const DownloadButton = ({ onClick }: { onClick: () => void }) => (
     </Button>
 )
 
-export const Toolbar = ({ selectedModels, onDeselectAll }: ToolbarProps) => {
+export const Toolbar = ({
+    selectedModels,
+    onDeselectAll,
+    downloadable = true,
+}: ToolbarProps) => {
     const [downloadDialogOpen, setDownloadDialogOpen] = useState(false)
 
     const isAnySelected = selectedModels.size > 0
 
-    const DownloadButtonElement = (
+    const DownloadButtonElement = downloadable ? (
         <DownloadButton onClick={() => setDownloadDialogOpen(true)} />
-    )
+    ) : undefined
```

Comment:
There's currently no conditional in `ToolbarSelected` - so currently this would render undefined. I think it's cleaner for this to be `null`, since that is valid to render.